dist/bu-blocks-frontend.js

```diff
@@ -602,14 +602,14 @@ const bu_blocks = {};
 		if ( ! group ) {
 			return blocks;
 		}
-		var containers = group.querySelectorAll( '.wp-block-bu-collapsible' );
+		var containers = group.querySelectorAll( '.js-wp-block-bu-collapsible' );
 
 		containers.forEach( function( element, i ) {
 			var block = {};
 
 			block.container = element;
-			block.toggle = element.querySelector( '.bu-block-collapsible-toggle' );
-			block.panel = element.querySelector( '.bu-block-collapsible-content' );
+			block.toggle = element.querySelector( '.js-bu-block-collapsible-toggle' );
+			block.panel = element.querySelector( '.js-bu-block-collapsible-content' );
 			blocks.push( block );
```

Comment:
This is really similar looking code to lines 581-585 - does it make sense to abstract the code around here to something like a getChildBlocks function?

Reply:
these are in two separate files for the collapsible block & collapsible control block. I think you are seeing it together because it's the compiled file. We don't have a mechanism to share functions for frontend code from one block to the next.